crates/uv/tests/it/pip_tree.rs

```diff
@@ -455,323 +430,21 @@ fn prune() {
         .arg("--cache-dir")
         .arg(context.cache_dir.path())
         .arg("--prune")
-        .arg("numpy")
+        .arg("werkzeug")
         .arg("--prune")
-        .arg("joblib")
+        .arg("jinja2")
         .env(EnvVars::VIRTUAL_ENV, context.venv.as_os_str())
         .env(EnvVars::UV_NO_WRAP, "1")
         .current_dir(&context.temp_dir), @r###"
     success: true
     exit_code: 0
     ----- stdout -----
-    scikit-learn v1.4.1.post1
-    ├── scipy v1.12.0
-    └── threadpoolctl v3.4.0
-
-    ----- stderr -----
-    "###
-    );
-
-    uv_snapshot!(context.filters(), Command::new(get_bin())
-        .arg("pip")
-        .arg("tree")
-        .arg("--cache-dir")
-        .arg(context.cache_dir.path())
-        .arg("--prune")
-        .arg("scipy")
-        .env(EnvVars::VIRTUAL_ENV, context.venv.as_os_str())
-        .env(EnvVars::UV_NO_WRAP, "1")
-        .current_dir(&context.temp_dir), @r###"
-    success: true
-    exit_code: 0
-    ----- stdout -----
-    scikit-learn v1.4.1.post1
-    ├── numpy v1.26.4
-    ├── joblib v1.3.2
-    └── threadpoolctl v3.4.0
-
-    ----- stderr -----
-    "###
-    );
-}
-
-#[test]
-#[cfg(target_os = "macos")]
-fn complex_nested_dependencies_inverted() {
```

Comment:
I don't think these "complex" tests are worth it. We have to install the whole tree here, and they aren't testing any new behavior.